tests/views_integration/Tribe/Events/Views/V2/Partials/Components/iCalTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\Components;
+
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class iCalTest extends HtmlPartialTestCase
```

Comment:
small thing, but in staying consistent with all the other tests, if our template is `ical-link.php`, our test class and file name should be `Ical_LinkTest`